plugin/trino-delta-lake/pom.xml

```diff
@@ -148,11 +186,29 @@
             <artifactId>guava</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.http-client</groupId>
+            <artifactId>google-http-client</artifactId>
+            <version>1.30.1</version>
```

Comment:
as https://github.com/trinodb/trino/pull/12523/files#r962920859 , should this version be provided by `google-api-client-bom`?

Reply:
https://github.com/googleapis/google-api-java-client/blob/v1.30.1/google-api-client-bom/pom.xml the BOM file is limited to only a few `com.google.api-client` dependencies.